libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupGoogleFragment.java

```diff
@@ -38,7 +36,7 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ((WordPress) getActivity().getApplication()).component().inject(this);
+        AndroidSupportInjection.inject(this);
```

Comment:
This injector call should be done in `onAttach()` instead.

Reply:
I moved the injector call in d8894f1.